src/main/java/org/mdbenefits/app/data/enums/MoneyOnHandType.java

```diff
@@ -0,0 +1,36 @@
+package org.mdbenefits.app.data.enums;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public enum MoneyOnHandType {
+    CHECKING("checking", "moneyonhand-types.checking-account"),
+    SAVINGS("savings", "moneyonhand-types.savings-account"),
+    BONDS("bonds", "moneyonhand-types.bonds"),
+    CASH_ON_HAND("cash", "moneyonhand-types.cash-on-hand"),
+    MUTUAL_FUNDS("mutual funds", "moneyonhand-types.mutual-funds"),
+    STOCKS("stocks", "moneyonhand-types.stocks");
+
+    private final String value;
+    private final String labelSrc;
+
+    static private final Map<String, MoneyOnHandType> MAP_BY_VALUE = new HashMap<>();
+
+    static {
+        for (MoneyOnHandType type : MoneyOnHandType.values()) {
+            MAP_BY_VALUE.put(type.value, type);
+        }
+    }
+
+    MoneyOnHandType(String value, String labelSrc) {
+        this.value = value;
+        this.labelSrc = labelSrc;
+    }
+
+    public static String getLabelSrc(String value) {
```

Comment:
When we talked I mentioned that this may not need to be static. I was confused because it looks like a getter for the field `labelSrc`.  But really this is doing a look up by value.  Could you rename this to be something like `lookupLabelSrcByValue` or, have it return the enum instead, and then pull the label src from it?  I don't have a preference for which you choose, just that this method name be a bit clearer and not collide with the getter name.

Reply:
@bseeger - I realize I don't need this method because there seems to be built in getters for value and labelSrc. So I removed the method altogether and confirmed that the code still works as expected.